main.dart

```diff
@@ -0,0 +1,46 @@
+import 'Family.dart';
+import 'FamilyWrapper.dart';
+main(List<String> args) {
+  var obj1 = Singleton();
+
+  print(obj1.addNode(1, "Saurabh Singh"));
```

Comment:
object variable can be named more intuitively, so that its clear by the name itself, what that variable is supposed to hold. example: `graph` instead of `obj1`